client/templates/discussion/all_discussions.js

```diff
@@ -0,0 +1,122 @@
+import _ from 'lodash';
+
+Template.allDiscussions.onCreated(function() {
+  const title = "CodeBuddies | Discussions";
+  const metaInfo = {
+    name: "description",
+    content: "CodeBuddies is a community of independent code learners who enjoy sharing knowledge and helping each other learn faster. We come from all over the world; there are members living in the United States, Japan, Sweden, the United Kingdom, Russia, Australia, Canada, India, and more. Everyone is welcome, independent of previous knowledge."
+  };
+
+  DocHead.setTitle(title);
+  DocHead.addMeta(metaInfo);
+
+  let instance = this;
+  instance.limit = new ReactiveVar(10);
+  instance.flag = new ReactiveVar(false);
+  instance.discussionFilter = new ReactiveVar('newest');
+
+  let projection = new Object();
```

Comment:
why not `projection = {}` ?

Reply:
use of `new` operator provides more readability.